presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -500,22 +500,20 @@ private PlanBuilder window(PlanBuilder subPlan, QuerySpecification node)
             Window window = windowFunction.getWindow().get();
 
             // Extract frame
-            WindowFrame.Type frameType = null;
-            FrameBound.Type frameStartType = null;
-            FrameBound.Type frameEndType = null;
-            Expression frameStart = null;
-            Expression frameEnd = null;
+            WindowFrame.Type frameType;
```

Comment:
Unnecessary change (should be squashed with previous commit). But if you apply my suggestion (from comment https://github.com/Teradata/presto/pull/304/commits/f5a57febdbef7b62f1ef628aaede4c40d666c32b#r74996131) it will be irrelevant anyway.